quarkchain/cluster/jsonrpc.py

```diff
@@ -871,16 +871,17 @@ def get_data_default(key, decoder, default=None):
         return self.counters
 
     @public_methods.add
-    async def gasPrice(self, full_shard_key: int):
+    async def gasPrice(self, full_shard_key: str, token_id: str):
         full_shard_key = shard_id_decoder(full_shard_key)
+        token_id = quantity_decoder(token_id)
```

Comment:
nit: no need for assignment, can directly use `quantity_decoder(token_id)` in line 884